src/pages/patientView/genomicOverview/tracksHelper.ts

```diff
@@ -204,7 +204,7 @@ export function plotMuts(p: any, config: any,chmInfo: any,row: any, mutations: A
         var mutObj: Mutation = mutObjs[i];
         if (typeof mutObj.gene.chromosome !== 'undefined') {
             var chm = translateChm(mutObj.gene.chromosome);
-            if (chm != null||chm <= chmInfo.hg19.length) {
+            if (chm != null && chm <= chmInfo.hg19.length) {
```

Comment:
@yichaoS Please verify that the new code is what you intended.  The old condition was always true unless `chm === undefined`.